app/assets/stylesheets/administrate/components/_date_time_picker.scss

```diff
@@ -1,3 +1,8 @@
 .form-field {
   position: relative;
 }
+
+.bootstrap-datetimepicker-widget table td, 
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Line contains trailing whitespace